### PR TITLE
chore: config v1.3.2

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**

### CODE REVIEW

File: src/form/common/default-validation-conditionals.ts [hunk 1]
```diff
@@ -253,51 +259,11 @@ export const brideOrGroomBirthDateValidators = (spouseType: string) => [
...
-    parameters: [3]
-  }
-] satisfies Validator[]
-
 export const exactDateOfBirthUnknownConditional = [
   {
     action: 'hide',
-    expression: '!values.exactDateOfBirthUnknown'
+    expression: '!values.customizedExactDateOfBirthUnknown'
+    // '!values.exactDateOfBirthUnknown || !values.customizedExactDateOfBirthUnknown || !values.birthInformantCustomizedExactDateOfBirthUnknown'
```
Comment (makelicious) [inline src/form/common/default-validation-conditionals.ts:266]: Are these still needed or can they be removed?

File: src/form/common/default-validation-conditionals.ts [hunk 2]
```diff
@@ -487,3 +462,64 @@ export const detailsDontExist = '!values.detailsExist'
 // primary address same as other primary
 export const primaryAddressSameAsOtherPrimaryAddress =
   'values.primaryAddressSameAsOtherPrimary'
+
+export const primaryAddressSameAsOtherPrimary: Conditional[] = [
+  {
+    action: 'hide',
+    expression: 'values.primaryAddressSameAsOtherPrimary'
```
Comment (makelicious) [inline src/form/common/default-validation-conditionals.ts:469]: can `primaryAddressSameAsOtherPrimaryAddress` be used?

File: src/login-config.prod.js [hunk 3]
```diff
@@ -12,9 +12,11 @@ window.config = {
...
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
+  COUNTRY: 'MDG',
+  LANGUAGES: 'fr,en,mg',
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
   CLIENT_APP_URL: 'https://register.{{hostname}}/',
   COUNTRY_CONFIG_URL: 'https://countryconfig.{{hostname}}',
-  SENTRY: '{{sentry}}'
+  SENTRY: '{{sentry}}',
+  LOGROCKET: ''
```
Comment (makelicious) [inline src/login-config.prod.js:21]: is logrocket in use anymore?

File: src/login-config.prod.js [hunk 4]
```diff
@@ -12,9 +12,11 @@ window.config = {
   AUTH_API_URL: 'https://gateway.{{hostname}}/auth/',
   CONFIG_API_URL: 'https://config.{{hostname}}',
   // Country code in uppercase ALPHA-3 format
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
+  COUNTRY: 'MDG',
+  LANGUAGES: 'fr,en,mg',
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
```
Comment (makelicious) [inline src/login-config.prod.js:17]: `AVAILABLE_LANGUAGES_SELECT` was this deprecated on `1.4`?

File: src/form/common/common-optional-fields.ts [hunk 5]
```diff
@@ -96,7 +97,7 @@ export const registrationEmail: SerializedFormField = {
   name: 'registrationEmail',
   type: 'TEXT',
   label: formMessageDescriptors.email,
-  required: true, // Email is the configured INFORMANT_NOTIFICATION_DELIVERY_METHOD in Farajaland
+  required: false, // Email is the configured INFORMANT_NOTIFICATION_DELIVERY_METHOD in Farajaland
```
Comment (makelicious) [inline src/form/common/common-optional-fields.ts:100]: Reference to farajaland, do we need it?

File: src/form/common/common-optional-fields.ts [hunk 6]
```diff
@@ -36,12 +36,13 @@ export const exactDateOfBirthUnknown = (
...
   },
-  hideInPreview: true,
+  hideInPreview: false, // true
   required: false,
   hideHeader: true,
   initialValue: false,
   validator: [],
-  conditionals: exactDobConditional.concat(conditionalCase),
+  conditionals: conditionalCase,
+  // conditionals: exactDobConditional.concat(conditionalCase),
```
Comment (makelicious) [inline src/form/common/common-optional-fields.ts:45]: is this needed?

File: src/form/common/common-optional-fields.ts [hunk 7]
```diff
@@ -36,12 +36,13 @@ export const exactDateOfBirthUnknown = (
     description: 'Checkbox for exact date of birth unknown',
     id: 'form.field.label.exactDateOfBirthUnknown'
   },
-  hideInPreview: true,
+  hideInPreview: false, // true
```
Comment (makelicious) [inline src/form/common/common-optional-fields.ts:39]: Remove extra comment

File: src/form/birth/required-fields.ts [hunk 8]
```diff
@@ -41,7 +41,7 @@ export const informantType: SerializedFormField = {
 }
 
 export const getPlaceOfBirthFields = (): SerializedFormField[] => [
-  divider('place-of-birth'),
+  //divider('place-of-birth'),
```
Comment (makelicious) [inline src/form/birth/required-fields.ts:44]: Can this be removed?

File: src/form/birth/index.ts [hunk 9]
```diff
@@ -19,25 +19,18 @@ import {
...
   getNationality,
   otherInformantType,
+  getNationalID,
   getDetailsExist,
   getReasonNotExisting
 } from '../common/common-required-fields'
 import {
-  exactDateOfBirthUnknown,
-  getAgeOfIndividualInYears,
+  // getAgeOfIndividualInYears,
```
Comment (makelicious) [inline src/form/birth/index.ts:27]: Can this be removed

File: src/form/birth/index.ts [hunk 10]
```diff
@@ -138,7 +139,7 @@ export const birthForm: ISerializedForm = {
                 },
                 {
                   defaultMessage:
-                    'Once the declaration is processed you will receive an SMS to tell you when to visit the office to collect the certificate - Take your ID with you.',
+                    'Once the declaration is processed you will receive you will receive an SMS to tell you when to visit the office to collect the certificate - Take your ID with you.',
```
Comment (makelicious) [inline src/form/birth/index.ts:142]: Let's use the one on 1.5. without duplicate `you will receive`

File: src/form/birth/index.ts [hunk 11]
```diff
@@ -170,31 +171,51 @@ export const birthForm: ISerializedForm = {
...
+                description: 'A form field that asks for name of fokontany',
+                defaultMessage: 'Fokontany'
+              }
+            ),
             // PLACE OF BIRTH FIELDS WILL RENDER HERE
-            divider('place-of-birth-seperator'),
-            attendantAtBirth,
-            birthType,
-            weightAtBirth
+            // divider('place-of-birth-seperator'),
```
Comment (makelicious) [inline src/form/birth/index.ts:213]: Are the commented out lines needed?

File: src/form/birth/index.ts [hunk 12]
```diff
@@ -215,20 +236,21 @@ export const birthForm: ISerializedForm = {
...
+              'informantID',
+              hideIfNidIntegrationEnabled.concat(hideIfInformantMotherOrFather),
+              getNationalIDValidators('informant'),
+              certificateHandlebars.informantNID
+            ),
+            getCustomizedExactDateOfBirthUnknown(
+              'informant',
+              hideIfInformantMotherOrFather
+            ),
+            // exactDateOfBirthUnknown(hideIfInformantMotherOrFather),
```
Comment (makelicious) [inline src/form/birth/index.ts:253]: we could remove this if it's not needd

File: src/form/birth/index.ts [hunk 13]
```diff
@@ -246,29 +268,35 @@ export const birthForm: ISerializedForm = {
...
               exactDateOfBirthUnknownConditional.concat(
                 hideIfInformantMotherOrFather
-              ),
-              ageOfIndividualValidators
+              )
             ),
-            getNationality(
-              certificateHandlebars.informantNationality,
-              hideIfInformantMotherOrFather
+            // getAgeOfIndividualInYears(
```
Comment (makelicious) [inline src/form/birth/index.ts:277]: uncomment or remove if not needed

File: src/form/birth/index.ts [hunk 14]
```diff
@@ -277,14 +305,23 @@ export const birthForm: ISerializedForm = {
               }
             ]),
             // ADDRESS FIELDS WILL RENDER HERE
-            divider('informant-address-seperator', [
+            /*  divider('informant-address-seperator', [
```
Comment (makelicious) [inline src/form/birth/index.ts:308]: uncomment or remove if not needed

File: src/form/birth/index.ts [hunk 15]
```diff
@@ -309,63 +346,65 @@ export const birthForm: ISerializedForm = {
...
-            getFamilyNameField(
-              'motherNameInEnglish',
-              motherFamilyNameConditionals,
-              certificateHandlebars.motherFamilyName
+            getNationality(
+              certificateHandlebars.motherNationality,
+              detailsExist
             ), // Required field.
+            getNationalID(
+              'iD',
```
Comment (makelicious) [inline src/form/birth/index.ts:354]: 'ID' or 'id' maybe?  let me know if intentionally `iD`

File: src/form/birth/index.ts [hunk 16]
```diff
@@ -309,63 +346,65 @@ export const birthForm: ISerializedForm = {
...
-            ...getIDNumberFields(
-              'mother',
-              hideIfNidIntegrationEnabled.concat(detailsExist),
-              true
-            ),
-            // preceding field of address fields
-            divider('mother-nid-seperator', detailsExist),
-            // ADDRESS FIELDS WILL RENDER HERE
-            divider('mother-address-seperator', detailsExist),
+            // exactDateOfBirthUnknown(detailsExistConditional),
```
Comment (makelicious) [inline src/form/birth/index.ts:373]: uncomment or remove if not needed

File: src/form/birth/index.ts [hunk 17]
```diff
@@ -309,63 +346,65 @@ export const birthForm: ISerializedForm = {
...
-            getEducation(certificateHandlebars.motherEducationalAttainment),
-            getOccupation(certificateHandlebars.motherOccupation, [
-              {
-                action: 'hide',
-                expression: '!values.detailsExist'
-              }
-            ]),
-            multipleBirth
+            getOccupation(certificateHandlebars.motherOccupation),
+            // getCustomAddress('mother', detailsExist),
```
Comment (makelicious) [inline src/form/birth/index.ts:396]: uncomment or remove if not needed

File: src/form/birth/index.ts [hunk 18]
```diff
@@ -398,62 +437,70 @@ export const birthForm: ISerializedForm = {
...
-            ...getIDNumberFields(
-              'father',
-              hideIfNidIntegrationEnabled.concat(detailsExist),
-              true
-            ),
-            // preceding field of address fields
-            divider('father-nid-seperator', detailsExist),
-            // ADDRESS FIELDS WILL RENDER HERE
-            divider('father-address-seperator', detailsExist),
+            // exactDateOfBirthUnknown(detailsExistConditional),
```
Comment (makelicious) [inline src/form/birth/index.ts:464]: uncomment or remove if not needed

File: src/form/birth/index.ts [hunk 19]
```diff
@@ -398,62 +437,70 @@ export const birthForm: ISerializedForm = {
...
             getMaritalStatus(certificateHandlebars.fatherMaritalStatus, [
               {
                 action: 'hide',
                 expression: '!values.detailsExist'
               }
             ]),
-            getEducation(certificateHandlebars.fatherEducationalAttainment),
-            getOccupation(certificateHandlebars.fatherOccupation, [
+            getOccupation(certificateHandlebars.fatherOccupation),
+            // getCustomAddress('father', detailsExist),
```
Comment (makelicious) [inline src/form/birth/index.ts:487]: uncomment or remove if not needed

File: src/form/birth/index.ts [hunk 20]
```diff
@@ -398,62 +437,70 @@ export const birthForm: ISerializedForm = {
...
-                expression: '!values.detailsExist'
+                id: 'form.field.label.customAddress',
+                description:
+                  'A form field that asks for father current address',
+                defaultMessage: 'Address'
               }
-            ])
+            ),
+            divider('father-address-seperator', detailsExist)
+            // getEducation(certificateHandlebars.fatherEducationalAttainment),
```
Comment (makelicious) [inline src/form/birth/index.ts:503]: uncomment or remove if not needed

File: src/form/birth/certificate-handlebars.ts [hunk 21]
```diff
@@ -17,38 +17,21 @@ export const certificateHandlebars = {
...
-  registrarSignature: 'registrarSignature', // @deprecated use registrar.signature instead
+  registrarName: 'registrarName',
+  role: 'role',
+  registrarSignature: 'registrarSignature',
   registrationDate: 'registrationDate',
-  registrationLocation: 'registrationLocation', // @deprecated use registrar.office/state/district instead
+  registrationLocation: 'registrationLocation',
   contactEmail: 'contactEmail',
   contactPhoneNumber: 'contactPhoneNumber',
+  mosipAid: 'mosipAid',
```
Comment (makelicious) [inline src/form/birth/certificate-handlebars.ts:27]: Could we unify the casing to be `AID` or `Aid` for both

File: src/form/birth/certificate-handlebars.ts [hunk 22]
```diff
@@ -17,38 +17,21 @@ export const certificateHandlebars = {
   certificateDate: 'certificateDate',
   registrar: 'registrar',
   registrationAgent: 'registrationAgent',
-  registrarName: 'registrarName', // @deprecated use registrar.name instead
-  role: 'role', // @deprecated use registrar.role instead
-  registrarSignature: 'registrarSignature', // @deprecated use registrar.signature instead
+  registrarName: 'registrarName',
```
Comment (makelicious) [inline src/form/birth/certificate-handlebars.ts:20]: Do the deprecated messages have relevance to us? Should they stay?

File: src/client-config.prod.js [hunk 23]
```diff
@@ -15,10 +15,12 @@ window.config = {
   AUTH_URL: 'https://gateway.{{hostname}}/auth/',
   MINIO_BUCKET: 'ocrvs',
   COUNTRY_CONFIG_URL: 'https://countryconfig.{{hostname}}',
-  // Country code in uppercase ALPHA-3 format
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
-  SENTRY: '{{sentry}}',
+  // Country code in uppercase ALPHA-3 format// Country code in uppercase ALPHA-3 format
```
Comment (makelicious) [inline src/client-config.prod.js:18]: Remove the duplicate line

File: src/login-config.js [hunk 24]
```diff
@@ -12,9 +12,11 @@ window.config = {
   AUTH_API_URL: 'http://localhost:7070/auth/',
   CONFIG_API_URL: 'http://localhost:2021',
   // Country code in uppercase ALPHA-3 format
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
+  COUNTRY: 'MDG',
+  LANGUAGES: 'fr,en,mg',
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
```
Comment (makelicious) [inline src/login-config.js:17]: AVAILABLE_LANGUAGES_SELECT was this deprecated on 1.4?

File: src/login-config.js [hunk 25]
```diff
@@ -12,9 +12,11 @@ window.config = {
...
   CLIENT_APP_URL: 'http://localhost:3000/',
   COUNTRY_CONFIG_URL: 'http://localhost:3040',
-  SENTRY: ''
+  SENTRY: '',
+  LOGROCKET: ''
```
Comment (makelicious) [inline src/login-config.js:21]: is `logrocket` used anymore?

File: src/api/application/application-config-default.ts [hunk 26]
```diff
@@ -1,34 +1,42 @@
...
+      DELAYED: 2000
     },
     PRINT_IN_ADVANCE: true
   },
   COUNTRY_LOGO: countryLogo,
   CURRENCY: {
-    languagesAndCountry: ['en-US'],
-    isoCode: 'USD'
+    isoCode: 'MGA',
+    languagesAndCountry: ['en-US'] // mg-MG
```
Comment (makelicious) [inline src/api/application/application-config-default.ts:23]: what does the comment mean?

File: src/api/application/application-config-default.ts [hunk 27]
```diff
@@ -1,34 +1,42 @@
...
+      DELAYED: 2000
     },
     PRINT_IN_ADVANCE: true
   },
-  PHONE_NUMBER_PATTERN: '^0(7|9)[0-9]{8}$',
+  PHONE_NUMBER_PATTERN: '^[1-9]{1}[0-9]{8}|0[0-9]{9}|[0-9]{13}$',
   NID_NUMBER_PATTERN: '^[0-9]{10}$',
   LOGIN_BACKGROUND: {
-    backgroundColor: '36304E'
+    backgroundImage: `data:image/jpg;base64,${fs
```
Comment (makelicious) [inline src/api/application/application-config-default.ts:36]: Could the base64 be replaced with something like this:

```
import { backgroundImage } from '@countryconfig/api/application/login-bg-mdg.jpg';

LOGIN_BACKGROUND: {
  backgroundImage: (`url(${backgroundImage})`),
  imageFit: 'FILL
}
```

File: src/client-config.js [hunk 28]
```diff
@@ -16,9 +16,12 @@ window.config = {
   MINIO_BUCKET: 'ocrvs',
   COUNTRY_CONFIG_URL: 'http://localhost:3040',
   // Country code in uppercase ALPHA-3 format
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
-  SENTRY: '',
+  COUNTRY: 'MDG',
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
```
Comment (makelicious) [inline src/client-config.js:20]: AVAILABLE_LANGUAGES_SELECT was this deprecated on 1.4?

File: src/client-config.js [hunk 29]
```diff
@@ -16,9 +16,12 @@ window.config = {
...
+  LANGUAGES: 'fr,en,mg',
+  SENTRY:
+    'https://f892d643aab642108f44e2d1795706bc@o309867.ingest.sentry.io/1774604',
+  LOGROCKET: '',
```
Comment (makelicious) [inline src/client-config.js:24]: Is logrocket still in use?

File: src/client-config.prod.js [hunk 30]
```diff
@@ -15,10 +15,12 @@ window.config = {
...
+  COUNTRY: 'MDG', // MDG
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
```
Comment (makelicious) [inline src/client-config.prod.js:20]: `AVAILABLE_LANGUAGES_SELECT` was this deprecated on 1.4?

**Review by makelicious — COMMENTED** [review]
Added comments for 'feature-agnostic' places

File: src/client-config.js (hunk 28 above)
Comment (tahmidrahman-dsi) [reply src/client-config.js:20]: Great catch! will remove it, thanks @makelicious

File: src/client-config.prod.js (hunk 30 above)
Comment (tahmidrahman-dsi) [reply src/client-config.prod.js:20]: Great catch! will remove it

File: src/client-config.js (hunk 29 above)
Comment (tahmidrahman-dsi) [reply src/client-config.js:24]: No it is not in use, I am going to remove

File: src/form/common/common-optional-fields.ts (hunk 5 above)
Comment (tahmidrahman-dsi) [reply src/form/common/common-optional-fields.ts:100]: No, we can get rid of the comments

File: src/form/common/default-validation-conditionals.ts (hunk 1 above)
Comment (ikotonirina) [reply src/form/common/default-validation-conditionals.ts:266]: Yeah
All commit that we've made from the v1.3.2 are still needed
But you can customize or refactor it.
I think we need to make a point or call about it

File: src/form/common/common-optional-fields.ts (hunk 6 above)
Comment (tahmidrahman-dsi) [reply src/form/common/common-optional-fields.ts:45]: nope

File: src/api/application/application-config-default.ts (hunk 26 above)
Comment (ikotonirina) [reply src/api/application/application-config-default.ts:23]: It is about the currency config,

File: src/form/birth/index.ts (hunk 15 above)
Comment (tahmidrahman-dsi) [reply src/form/birth/index.ts:354]: unfortunately yes, there are several places where we are maintaining this case 😞

File: src/form/birth/certificate-handlebars.ts (hunk 22 above)
Comment (tahmidrahman-dsi) [reply src/form/birth/certificate-handlebars.ts:20]: sorry deprecate messages shoud stay, getting those back

File: src/login-config.js (hunk 25 above)
Comment (tahmidrahman-dsi) [reply src/login-config.js:21]: nope

File: src/api/application/application-config-default.ts (hunk 27 above)
Comment (tahmidrahman-dsi) [reply src/api/application/application-config-default.ts:36]: yup that can be done

File: src/login-config.prod.js (hunk 3 above)
Comment (tahmidrahman-dsi) [reply src/login-config.prod.js:21]: nope

File: src/form/birth/certificate-handlebars.ts (hunk 21 above)
Comment (tahmidrahman-dsi) [reply src/form/birth/certificate-handlebars.ts:27]: we can get rid of mosip related stuff if mosip is not applicable in Madagascar context. What do you think @makelicious ?

File: src/api/application/application-config-default.ts (hunk 27 above)
Comment (tahmidrahman-dsi) [reply src/api/application/application-config-default.ts:36]: Generally I think that would be a very good improvement. But after a bit of trying, seems we might have to adjust several places. Best if we had a public endpoint which would only serve this image. However, all this change might be a improvement for general use case but may be a bit large under the Madagascar configuration scope

**Pull request merged**